### PR TITLE
removed my name from the file

### PR DESCRIPTION
**Pull request by pranav-pandey0804**

### CODE REVIEW

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm - you just want to remove your emails form this file, or maybe all files, or maybe even more - add them to "excluded" list so they will never appear?

**Comment by kundan2707** [comment]
@pranav-pandey0804 you also need to remove your name from https://github.com/cncf/gitdm/blob/0d8ccd024131b4d07112079eea727547f715ec24/developers_affiliations6.txt#L4517

**Comment by gmaanos** [comment]
Instead of removing the entry, you need to add the end date. commenting inline

File: company_developers8.txt [hunk 1]
```diff
@@ -9556,7 +9556,6 @@ NEC Corporation:
 	oomichi: ken-oomichi!wx.jp.nec.com, oomichi!users.noreply.github.com until 2022-06-01
 	parauliya: parauliya!users.noreply.github.com until 2017-01-01
 	poothia: poothia!users.noreply.github.com, poothia.shubhendu!gmail.com, shubhendu.poothia!india.nec.com until 2019-08-01
-	pranav-pandey0804: pranav-pandey0804!users.noreply.github.com, pranav.pandey!india.nec.com
```
Comment (gmaanos) [inline company_developers8.txt:9559]: you should not remove the entry instead add the end date in all the file

pranav-pandey0804: pranav-pandey0804!users.noreply.github.com, pranav.pandey!india.nec.com
	NEC Corporation until 2024-11-08
	Independent from 2024-11-09

Example- https://github.com/cncf/gitdm/commit/e87b8162c9cbe8370611eacf4e99c117b39166b6

**Review by gmaanos — COMMENTED** [review]
please update all other file also

**Comment by lukaszgryglicki** [comment]
Right, if you just changed the employer and want to update - then you can follow example of other PRs or just give me a list of employers with start/end dates and I will do it for you.
If you wanted to remove your email or GitHub ID you can also request to remove it from all files within all `gitdm/devstats` repos + add them to the exclusion list so I can remove them, in that case create a PR according to this: https://github.com/cncf/gitdm/blob/master/FORBIDDEN_DATA.md.

**Comment by gmaanos** [comment]
@lukaszgryglicki  Thanks, Pranav worked in NEC until 2024-11-08 (8th Nov 2024)  and he would like to update the same in here. I think if you or he can update his affiliation "NEC Corporation until 2024-11-08" it will do the required updates.

**Comment by lukaszgryglicki** [comment]
OK will do such an update on my next CNCF working day.

File: company_developers8.txt (hunk 1 above)
Comment (lukaszgryglicki) [reply company_developers8.txt:9559]: I will handle this manually.

**Comment by lukaszgryglicki** [comment]
This is done, also updated `cncf/devstats`.

**Pull request closed without merging**